app/components/Admin/ProposedService.js

```diff
@@ -86,6 +95,16 @@ class ProposedService extends React.Component {
     }
     return scheduleOutput;
   }
+
+  renderCategoryFields() {
+    return (
+      <div key={"cat"} className="request-entry">
+        <p className="request-cell name">Categories</p>
+        <textarea readOnly className="request-cell value" value={this.state.categories} />
```

Comment:
So this is readonly? That's probably fine for now, but we should let the data team know that they won't be able to edit the proposed categories.